src/core/etl/tests/Flow/ETL/Tests/Unit/Extractor/FlowTestCase.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Flow\ETL\Tests\Unit\Extractor;
+
+use Flow\ETL\{Config, Extractor, FlowContext, Rows};
+use PHPUnit\Framework\TestCase;
+
+abstract class FlowTestCase extends TestCase
+{
+    public function assertExtractorCountBatches(int $expectedCount, Extractor $extractor) : void
+    {
+        static::assertCount(
+            $expectedCount,
+            \iterator_to_array($extractor->extract(new FlowContext(Config::default())))
+        );
+    }
+
+    public function assertExtractorCountRows(int $expectedCount, Extractor $extractor) : void
```

Comment:
`assertExtractedRowsCount`

Reply:
Fixed in https://github.com/flow-php/flow/pull/1291/commits/87f5a00d47f4a0842286506070275c55b786f64f